cli/cmdlineparser.cpp

```diff
@@ -864,6 +863,12 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
             // Filter errors
             else if (std::strncmp(argv[i], "--suppressions-list=", 20) == 0) {
                 std::string filename = argv[i]+20;
+                
+                if (filename.empty()) {
```

Comment:
This check is kind of redundant since the code below essentially provides this information - albeit just implicitly.